server/services/bodyScanInfoDecorator.ts

```diff
@@ -27,9 +28,9 @@ export class BodyScanInfoDecorator {
 
   public async decorateSingle<T extends HasPrisonNumber>(item: T): Promise<WithBodyScanInfo<T>> {
     const token = await this.hmppsAuthClient.getSystemClientToken()
-    const { numberOfBodyScans, numberOfBodyScansRemaining } = await this.bodyScanClientFactory(
+    const { numberOfBodyScans, numberOfBodyScansRemaining, bodyScanStatus } = await this.bodyScanClientFactory(
       token
     ).getSingleBodyScanInfo(item.prisonNumber)
-    return { ...item, numberOfBodyScans, numberOfBodyScansRemaining }
+    return { ...item, numberOfBodyScans, numberOfBodyScansRemaining, bodyScanStatus }
```

Comment:
`bodyScanStatus` will determine if we show body scan approaching limit warning so added to decorator so available to summary page templates